### PR TITLE
#712: Change CreateProjectPage to use NER Buttons

### LINKED ISSUE #712: [Frontend] - Change CreateProjectPage to use NER Buttons
### Description

In the pursuit of making the looking and feel of our app consistent, we wanna use new components that have been made (NERSuccessButton/NERFailButton) in every place where we currently have a cancel/save button combo. By doing this, we not only make all these save/cancel buttons adhere to the new look, but we also allow ourselves to change how all these save buttons look in the future by changing only one file!

For this ticket, change the save/cancel buttons in the CreateProjectPage component to use the NERSuccessButton/NERFailButton component rather than just the plain MUI button.

Put the cancel on the left, and the submit on the right

### Acceptance Criteria

The save/cancel buttons on CreateProjectPage use NERSuccessButton/NERFailButton and not Button
The save/cancel buttons on that page looks like the new success/cancel buttons (green and red contained, see the edit members section on a teams specific page to see a good example of how it looks).
Verify nothing looks weird/ everything works as it did before.
Buttons should be 2 px apart, good example: https://github.com/Northeastern-Electric-Racing/FinishLine/pull/731

### Proposed Solution

All the props should be the same, all you need to do is swap out the old for the new / for the Save/Cancel buttons respectively.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by vonbuelow**
## Changes

Changed the save/cancel buttons in the CreateProjectPage component to use the NERSuccessButton/NERFailButton component rather than just the plain MUI button. Cancel on the left and submit on the right. 

## Test Cases

- yarn test:frontend

## Screenshots

image

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #712

### CODE REVIEW

**Review by tigerhippo — COMMENTED** [review]
The code looks fine, just send me a screenshot of your tests failing.

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]

File: src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx [hunk 1]
```diff
@@ -148,12 +148,12 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
...
         <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
-            Create
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
+          <NERFailButton variant="contained" onClick={onCancel} sx={{ mx: 1 }}>
             Cancel
-          </Button>
+          </NERFailButton>
+          <NERSuccessButton variant="contained" type="submit" sx={{ mx: 1 }}>
```
Comment (Ari-S-123) [inline src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx:154]: `disabled={!allowSubmit}`

**Review by Ari-S-123 — CHANGES_REQUESTED** [review]

**Review by Ari-S-123 — APPROVED** [review]

File: src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx [hunk 2]
```diff
@@ -148,12 +148,12 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
           </Grid>
         </Grid>
         <Box display="flex" gap={2} sx={{ mt: 2 }}>
```
Comment (RChandler234) [inline src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx:150]: remove gap = {2} here it's adding another 2 px gap between the buttons, so it's 4 px apart total

File: src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx [hunk 3]
```diff
@@ -148,12 +148,12 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
...
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
+          <NERFailButton variant="contained" onClick={onCancel} sx={{ mx: 1 }}>
             Cancel
-          </Button>
+          </NERFailButton>
+          <NERSuccessButton variant="contained" type="submit" disabled={!allowSubmit} sx={{ mx: 1 }}>
+            Create
+          </NERSuccessButton>
         </Box>
```
Comment (RChandler234) [inline src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx:157]: these buttons should be in the bottom right corner of the form to be consistent with the rest of the forms

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by Ari-S-123 — DISMISSED** [review]

File: src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx [hunk 4]
```diff
@@ -147,14 +147,16 @@ const CreateProjectFormView: React.FC<CreateProjectFormViewProps> = ({ allowSubm
...
-        <Box display="flex" gap={2} sx={{ mt: 2 }}>
-          <SubmitButton variant="contained" color="primary" type="submit" disabled={!allowSubmit}>
-            Create
-          </SubmitButton>
-          <Button variant="outlined" color="secondary" onClick={onCancel}>
-            Cancel
-          </Button>
-        </Box>
+        <Grid container justifyContent="flex-end">
+          <Box display="flex" sx={{ mt: 2 }}>
```
Comment (RChandler234) [inline src/frontend/src/pages/CreateProjectPage/CreateProjectFormView.tsx:151]: you don't need the grid, just put the justifyContent="flex-end" on the Box component, and it should do the same thing

**Review by RChandler234 — CHANGES_REQUESTED** [review]

**Review by RChandler234 — APPROVED** [review]
LGTM!!

**Pull request merged**